app/javascript/tests/close_contact/CloseContactTable.test.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
```

Comment:
this file is missing a good amount of tests.  Check out `AssessmentTable.test.js` for a model of what should be tested

Reply:
Thanks! I stubbed it out with the intention of coming back to it, and forgot to 🙃